### PR TITLE
Enumerate rounds in dashboard front page

### PR DESCRIPTION
**Pull request by ryscheng**

### CODE REVIEW

**Review by jmoggr — APPROVED** [review]

**Comment by ryscheng** [comment]
Would it be reasonable to let the PR owner merge when ready? Just in case they have minor adjustments/fixes they need to make before merging?

**Comment by ryscheng** [comment]
Ya reasonable, I think the proposal specifically yes:
- Reviewer approves
- Author can still make changes (automatically approved) - we should roughly stick to really minor changes (see examples below)
- Author merges themself.

Use cases I can think of are:
- Author finds a typo, wants to just fix it real quick without turning it into a PR
- Author rebases and realizes there's some weird artifacts (e.g. weird ordering of sections on a page or something) and wants to fix real quick
- Basically reducing the scope to clearly minor fixes that are unambiguously reasonable to add in real quick. Anything major should be its own PR

I definitely overall agree with your suggestion towards frequent commits and PRs and lightweight.

**Comment by jmoggr** [comment]
That makes sense. I recall now Jorge mentioned that for code PRs in PL, traditionally the author merges it after approval.

I agree that we should adopt this method.

Personally the fire-and-forget nature of the alternative process for PRs appeals to me, but I see how this can be beneficial with little additional overhead.

**Comment by ccerv1** [comment]
@jmoggr @ryscheng can this be closed out? If not, please update with what's required next.

**Comment by ryscheng** [comment]
This is a pull request that was merged back in December (already closed). I believe this is unrelated to hypercerts, I was just kinda demoing to Jason how I'd implement impact evaluators in Plasmic. I removed it from the project and milestone
